### PR TITLE
Create source package instead

### PR DESCRIPTION
**Pull request by bording**
This makes the package produced include source files instead of an assembly that needs to be then be merged back in.

### CODE REVIEW

**Comment by Scooletz** [comment]
This should be reviewed as soon as there's some clarification provided for using `source shipping` instead of ILRepack https://github.com/Particular/NServiceBus.Metrics.ServiceControl/pull/6#issuecomment-336404716

File: src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.approved.txt [hunk 1]
```diff
@@ -1,42 +1,2 @@
 ﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleToAttribute("ServiceControl.Monitoring.Data.Tests")]
-[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
```
Comment (Scooletz) [inline src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.approved.txt]: Can you bump up the major @bording ? It's a totally different package after all :)

**Comment by Scooletz** [comment]
Ping @Particular/servicecontrol-maintainers Could you review and possibly merge it as soon as we have answer from @bording related to https://github.com/Particular/ServiceControl.Monitoring.Data/pull/9#issuecomment-336405168 ?

File: src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.approved.txt (hunk 1 above)
Comment (bording) [reply src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.approved.txt]: Done!

**Comment by bording** [comment]
>This should be reviewed as soon as there's some clarification provided for using source shipping instead of ILRepack Particular/NServiceBus.Metrics.ServiceControl#6 (comment)

https://github.com/Particular/NServiceBus.Metrics.ServiceControl/pull/6#issuecomment-337043858

**Pull request merged**